pkgs/tools/misc/chrome-export/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub
+, python
+}:
+
+stdenv.mkDerivation rec {
+  name = "chrome-export";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "bdesham";
+    repo = name;
+    rev = "v${version}";
+    sha256 = "0p1914wfjggjavw7a0dh2nb7z97z3wrkwrpwxkdc2pj5w5lv405m";
+  };
+
+  buildInputs = [ python ];
+
+  dontBuild = true;
+  installPhase = ''
+    mkdir -p $out/bin
+    cp export-chrome-bookmarks export-chrome-history $out/bin
+    mkdir -p $out/share/man/man1
+    cp man_pages/*.1 $out/share/man/man1
+  '';
+  doInstallCheck = true;
+  installCheckPhase = ''
+    bash test/run_tests $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = ''Scripts to convert Google Chrome's bookmarks and history to
```

Comment:
Please keep ``description`` one line only. Use ``longDescription`` to extend the meta.description to more lines if needed.

Reply:
Fixed.